analyses/optimus-acceptance-report/optimus-barcode-correction/barcode_correction_stats.py

```diff
@@ -2,26 +2,24 @@
 import sys
 import argparse
 
+
 # python3  script.py BAM  whitelist
-   
 
```

Comment:
Please add a description of what this script does here in comments. Copying the description below should be sufficient.